app/validators/preprint-form-validator.js

```diff
@@ -0,0 +1,38 @@
+import { validator, buildValidations } from 'ember-cp-validations';
+
+const Validations = buildValidations({
+    title: {
+        description: 'Title',
+        validators: [
+            validator('presence', true),
+            validator('length', {
+                // minimum length for title?
+                max: 300,
+            })
+        ]
+    },
+    abstract: {
+        description: 'Abstract',
+        validators: [
+            validator('presence', true),
+            validator('length', {
+                // currently min of 20 characters -- this is what arXiv has as the minimum length of an abstract
+                min: 20,
+                max: 5000
```

Comment:
I'm actually not sure that OSF descriptions have a maximum length at all. It's possible the preprint service will have different design restrictions, though.